core/src/main/java/org/infinispan/jmx/ResourceDMBean.java

```diff
@@ -40,7 +42,7 @@
  * @author Galder Zamarreño
  * @since 4.0
  */
-public final class ResourceDMBean implements DynamicMBean {
+final class ResourceDMBean implements DynamicMBean {
```

Comment:
The constructor should also be package-private

Reply:
I will be. But ... ResourceDMBean is affected by another PR, which makes the class PUBLIC and this constructor PACKAGE private :)))